src/Illuminate/Support/Env.php

```diff
@@ -128,11 +128,7 @@ public static function string(string $key, $default = null): string
     {
         $value = Env::get($key, $default);
 
-        if (! is_string($value)) {
-            throw new InvalidArgumentException(
-                sprintf('Environment value for key [%s] must be a string, %s given.', $key, gettype($value))
-            );
-        }
+        check_type($value, 'string', $key, 'Environment');
 
         return $value;
```

Comment:
You could also simplify this even further:
```suggestion
        return check_type($value, 'string', $key, 'Environment');
```
or
```suggestion
        return check_type(
            Env::get($key, $default),
            'string',
            $key,
            'Environment'
        );
```

Reply:
Thank you for the code review, this really looks cleaner.